crates/rolldown/tests/rolldown/misc/require_deconflict/artifacts.snap

```diff
@@ -11,7 +11,7 @@ snapshot_kind: text
 
 //#region main.js
 const __require = "test";
-var main_default = () => __require$1("test-dep") || __require$1("test-dep666");
+var main_default = () => __require$1("test-dep") || __require$1(`test-dep${666}`);
```

Comment:
We can recover the behavior with a bit of more code. @sapphi-red

Reply:
I am not sure what do you mean, but this is expected.